app/containers/SwimlaneContainer.js

```diff
@@ -111,6 +115,9 @@ const mergeProps = (stateProps, dispatchProps, ownProps) => {
     },
     onTaskUpdated: (task, updateAsana) => {
       dispatchProps.onTaskUpdated(task, id, currentProjectId, updateAsana);
+    },
+    onSwimlaneToggle: (id, collapsed) => {
```

Comment:
Is this still unnecessary? I don't think I fully understand what mergeProps and mapDispatchToProps do.